changelog/fix-wpml-save-course-lesson-bug

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Disable translation of Sensei post types, taxonomies and fields to temporary fix compatibilty issue.
```

Comment:
I think we should consider a different change log entry here, as this is likely to scare everyone with translated sites. It should mention that it's a compatibility issue with WPML.